overlord/hookstate/ctlcmd/services.go

```diff
@@ -115,6 +122,7 @@ func (c *servicesCommand) Execute([]string) error {
 	w := tabwriter.NewWriter(c.stdout, 5, 3, 2, ' ', 0)
 	defer w.Flush()
 
+	// XXX should service names be remapped back from snap-name_key to snap-name if needed?
```

Comment:
I would be cautious about doing this if the output is in someway parsed and the hooks expects the `snap-name.svc` instead of `snap-name_key.svc`. I don't know if this is something we need to consider. I think for the normal output (i.e `snap services`) it definitely needs to be `snap-name_key`, but for `snapctl` it should not need to consider that there may be additional instances as the services we query there will always be local to the snap

Reply:
what I meant here is whether it would make sense to remap snap-name_key -> snap-name in the output, if snap-name was used in the input arguments. I dug into enterprise-proxy, and they for instance, run `snapctl service <snap-name.svc>` for every service and only look for `'enabled'` in the output. At least in this case, remapping would not change the check's outcome.